src/sections/Header.js

```diff
@@ -14,27 +15,48 @@ import './Header.css';
 const Header = ({ isAuthenticated, user }) => (
   <AppBar position="fixed">
     <Toolbar>
-      <Link to="/">
-        <img className="Header-logo" src={logo} alt="code for denver logo" />
-      </Link>
-      <Grid container justify="space-between" alignItems="center">
-        {isAuthenticated && (
-          <Grid item>
-            <Link className="Header-link" to="/volunteers">
-              All Users
-            </Link>
-            <Link className="Header-link" to="/projects">
-              All Projects
-            </Link>
-            {userIsAdmin(user) && (
-              <Link className="Header-link" to="/admin">
-                Admin Resources
-              </Link>
-            )}
+      <Grid container justify="center" alignItems="center">
+        <Grid item sx={2} sm={1}>
+          <Link to="/">
+            <img
+              className="Header-logo"
+              src={logo}
+              alt="code for denver logo"
+            />
+          </Link>
+        </Grid>
+        <Grid item xs={10} sm={7}>
+          {isAuthenticated && (
+            <Grid container justify="flex-start" alignItems="center">
+              <Grid item sx={3}>
+                <Link className="Header-link" to="/volunteers">
+                  <Button color="secondary">ALL USERS</Button>
+                </Link>
+              </Grid>
+              <Grid item sx={3}>
+                <Link className="Header-link" to="/projects">
+                  <Button color="secondary">ALL PROJECTS</Button>
```

Comment:
Let's instead use the `component` prop on here to make it a link for better accessibility and less markup. See https://material-ui-next.com/demos/buttons/#third-party-routing-library

Reply:
I changed the buttons back to Links from the master branch,  The buttons have different spacing than the link components which is making the header much bigger.